content/docs/glossary/decentralized-exchange.mdx

```diff
@@ -3,41 +3,42 @@ title: Decentralized Exchange
+When creating a "sell"/"offer"/"ask" order in Stellar via the [Manage Sell Offer](../start/list-of-operations.mdx#manage-sell-offer) operation, the price is specified as 1 unit of base currency (the asset being sold), in terms of the quote asset (the asset that is being bought). For example, if you're _selling_ 100 XLM in exchange for 40 USD, you would specify the price as `{40, 100}`, which would be the equivalent of 2.5 XLM for 1 USD (or $.40 per XLM) (_nice profit_).
 
 #### Fees
 
-It's important to note that the price you set is unrelated to the fee you pay for submitting the offer as a part of a transaction. Fees are always paid in the native currency of the network (lumens), and are related to the transaction that you submit to the network (which contains your offer operation) as opposed to your offer itself.
+It's important to note that the price you set is unrelated to the fee you pay for submitting the order as a part of a transaction. Fees are always paid in the native currency of the network (lumens), and are related to the transaction that you submit to the network (which contains your order operation) as opposed to your order itself.
 
 For more information, take a look at [our guide on fees in Stellar](./fees.mdx).
 
-## Passive Offers
+## Passive Order
 
-**Passive offers** allow markets to have zero spread. If you want to offer USD from anchor A for USD from anchor B at a 1:1 price, you can create two passive offers so the two offers don't fill each other.
+**Passive orders** allow markets to have zero spread. If you want to exchange USD from anchor A for USD from anchor B at a 1:1 price, you can create two passive orders so the two orders don't fill each other. (will this actually be a zero spread? it should still be a tick wide right? or would it be a "locked market", where the bid and the ask is at the same price?)
 
-A passive offer is an offer that does not take a counteroffer of equal price. It will only fill if the prices are not equal. For example, if the best offer to buy BTC for XLM has a price of 100XLM/BTC, and you make a passive offer to sell BTC at 100XLM/BTC, your passive offer _does not_ take that existing offer. If you instead make a passive offer to sell BTC at 99XLM/BTC it would cross the existing offer and fill at 100XLM/BTC.
+A passive order is an order that does not execute against a marketable counter order with the same price. It will only fill if the prices are not equal. For example, if the best order to buy BTC for XLM has a price of 100XLM/BTC, and you make a passive offer to sell BTC at 100XLM/BTC, your passive offer _does not_ take that existing offer. If you instead make a passive offer to sell BTC at 99XLM/BTC it would cross the existing offer and fill at 100XLM/BTC. (i have some questions here...why does it not execute at a locked market (i.e. equal price) but at a crossed market it will execute?
```

Comment:
Flagging this last sentence too, so we don't push a question. I don't know the details of this, but I think either @nikhilsaraf or @jonjove will know.

Reply:
if this is similar to maker/taker orders it should read something like this:

A passive order is a limit order that does not immediately execute against a marketable counter order on the orderbook. It adds liquidity to the orderbook and will only be filled if another order non-passive order trades against it. For example, if there is an ask to sell 1 XLM at $0.10, and you create a passive buy order to buy to buy 1 XLM for $0.10 (or above), your passive buy order _does not_ take that existing $0.10 ask. Instead, it will place your bid at the highest possible price up to your limit price without executing the order, in this case ($0.09999999). It is a passive order because it will never initiate an order execution. It is an order that adds liquidity to and another order has to be initiated against it for it to execute.

Reply:
@phil-stellar This isn't the same as maker/taker orders. The behavior of create passive offer is not post only.